client/src/constants/route.ts

```diff
@@ -0,0 +1,13 @@
+import {Route} from 'next';
+import {URLS} from './urls';
+
+export const route = {
+  goWiki: (uuid: string) => `${URLS.wiki}/${uuid}` as Route,
+  goWikiEdit: (uuid: string) => `${URLS.wiki}/${uuid}${URLS.edit}` as Route,
+  goWikiLogs: (uuid: string) => `${URLS.wiki}/${uuid}${URLS.logs}` as Route,
+  goWikilog: (uuid: string, id: number) => `${URLS.wiki}/${uuid}/log/${id}` as Route,
+  goWikiWrite: () => `${URLS.wiki}${URLS.post}` as Route,
+  goDaemoon: () => `${URLS.wiki}/${URLS.daemoon}` as Route,
+
+  goAdminLogin: () => '/admin/login' as Route,
```

Comment:
하드코딩된 부분이 있어서 상수로 변경해주시면 될 것 같아요.

```suggestion
  goAdminLogin: () => URLS.login as Route,
```

추가로 확인해보니 LoginForm.tsx 파일의 63번째 줄인 `router.replace('/admin/documents');` 에서 하드코딩된 url을 사용하고 있어서 
`router.replace(route.goAdminDocuments());` 이런 식으로 수정이 필요할 것 같아요!